neuraxle/pipeline.py

```diff
@@ -272,7 +274,7 @@ def _load_checkpoint(self, data_container: DataContainer, context: ExecutionCont
         return self.steps_as_tuple, data_container
 
 
-class ResumablePipeline(Pipeline, ResumableStepMixin):
+class ResumablePipeline(Pipeline, ResumableMixin):
```

Comment:
The order of the base classes feels wrong. 

I'd have guessed it would be `class ResumablePipeline(ResumableMixin, Pipeline)` if you asked me to guess. That is for the ResumableMixin class' methods override the Pipeline class' methods, and not the other way around.